x-pack/plugins/stack_connectors/public/connector_types/inference/constants.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SUB_ACTION } from '../../../common/inference/constants';
+
+export const DEFAULT_CHAT_COMPLETE_BODY = {
+  input: 'What is Elastic?',
+};
+
+export const DEFAULT_RERANK_BODY = {
+  input: ['luke', 'like', 'leia', 'chewy', 'r2d2', 'star', 'wars'],
+  query: 'star wars main character',
+};
+
+export const DEFAULT_SPARSE_EMBEDDING_BODY = {
+  input: 'The sky above the port was the color of television tuned to a dead channel.',
+};
+
+export const DEFAULT_TEXT_EMBEDDING_BODY = {
+  input: 'The sky above the port was the color of television tuned to a dead channel.',
```

Comment:
Despite being API examples, I've been asked before to make connector example content relevant to the Kibana product. Not sure if Product will eventually ask the same of you in this case but just a heads up.